### DISCUSSION TITLE
Display to other collaborators that a person is currently reviewing a PR

### DISCUSSION THREAD

**Discussion by SimonMorkSaetre** [community/community#106313] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

Greetings. I am a developer in a small team where we use Github. We have experienced several times that multiple people review at the same time, where one approves while the other one is still reviewing or have forgot to submit the review. This causes issues where one of the developers finds an issue, but the PR is merged before they are able to submit the review.
It would be nice to have some kind of visual indicator or even an optional blocker to let everyone know that someone is currently having a look at the pull request.

**Comment by SergioMunozAnpworks** [comment]
Hello @SimonMorkSaetre,

My personal recommendation is to use teams on GitHub, among other advantages, it allow you to automatically assign the reviewer.
You can get documentation [it here](https://docs.github.com/en/organizations/organizing-members-into-teams/about-teams).
